Makefile

```diff
@@ -0,0 +1,7 @@
+# Rasterio make file.
+
+SHELL = /bin/bash
+
+wheels: Dockerfile.wheels build-linux-wheels.sh
+	docker build -f Dockerfile.wheels -t rasterio-wheelbuilder .
+	docker run -v `pwd`:/io rasterio-wheelbuilder
```

Comment:
Docker disk space is a constant battle for me and I've found adding `--rm` to every docker run command to be a good way to keep things clean. Not sure what the best practice is here.